documentation/src/docs/streamcuts.md

```diff
@@ -29,22 +29,22 @@ events can be continuously added to the stream and the `StreamCut` pointing to t
 newer events would have a different value. Similarly the `StreamCut` representing the head of the
 stream (with the oldest event) is an ever changing one as the stream retention policy could truncate the stream
 and the `StreamCut` pointing to the head of the stream post truncation would have a different value.
-```StreamCut.UNBOUNDED``` is used to represent such a position in the stream and the user can use it to
+`StreamCut.UNBOUNDED` is used to represent such a position in the stream and the user can use it to
 specify this ever changing stream position (both head and tail of the stream).
 
 It should be noted that `StreamCut`s obtained using the streaming client and batch client can be used
 interchangeably.
 
 ## StreamCut with Reader
 
-A ReaderGroup is a named collection of Readers that together, in parallel, read Events from a given Stream. Every
-Reader is always associated with a ReaderGroup. `StreamCut`(s) can be obtained from a ReaderGroup using the
-following API ```io.pravega.client.stream.ReaderGroup.getStreamCuts ```. This API returns a
-```Map<Stream, StreamCut>``` which represents the last known position of the Readers for all the streams managed by the ReaderGroup.
+A Reader Group is a named collection of Readers that together, in parallel, read Events from a given stream. Every
+Reader is always associated with a Reader Group. `StreamCut`(s) can be obtained from a Reader Group using the
+following API `io.pravega.client.stream.ReaderGroup.getStreamCuts `. This API returns a
+`Map<Stream, StreamCut>` which represents the **last known Position of the Readers** for all the streams managed by the Reader Group.
```

Comment:
Perhaps we could add the `generateStreamCuts()` api here. There are two ways to fetch `StreamCut`s pointing to the reader position.

Reply:
done